pkgs/applications/editors/vscode/extensions/ms-dotnettools.csdevkit/default.nix

```diff
@@ -0,0 +1,117 @@
+{ lib
+, vscode-utils
+, patchelf
+, icu
+, stdenv
+, openssl
```

Comment:
Would you mind ordering these by alphabetical order too?

Reply:
Done